sdk/tests/AccountBalanceQueryTest.cc

```diff
@@ -87,3 +90,23 @@ TEST_F(AccountBalanceQueryTest, ResetMutuallyExclusiveIds)
 
   EXPECT_FALSE(query.getContractId());
 }
+
+// Tests invoking of method execute() from AccountBalanceQuery.
+TEST_F(AccountBalanceQueryTest, ExecuteRequestToTestnetNode)
+{
+  // Given
+  const auto testAccountId = getTestAccountId();
+  Client testnetClient = Client::forTestnet();
+  testnetClient.setOperator(getTestAccountId(), ED25519PrivateKey::generatePrivateKey());
+
+  AccountBalanceQuery testAccountBalanceQuery;
+  testAccountBalanceQuery.setNodeAccountIds({ testAccountId });
```

Comment:
You don't need to set this here; the `setNodeAccountIds` function specifies which nodes to send this query to, not which accounts its querying.

Reply:
Thanks for noticing it. Using of `setNodeAccountIds` was removed in 98edae3efa6fad93a4be247eee3f09ab8c256b7e.